tests/storage/outofprocess_test.py

```diff
@@ -61,13 +62,11 @@ def oop_cleanup():
     ids=str,
 )
 def user_mount(request):
-    storage = request.param
-    if not storage.exists():
-        pytest.xfail("{} storage not available".format(storage.name))
-    tmpdir = os.path.join(storage.path, "tmp")
-    os.mkdir(tmpdir)
-    yield tmpdir
-    shutil.rmtree(tmpdir)
+    with Backend(request.param) as backend:
+        tmpdir = os.path.join(backend.path, "tmp")
+        os.mkdir(tmpdir)
+        yield tmpdir
+        shutil.rmtree(tmpdir)
```

Comment:
This looks like generic helper that many tests could use - providing a temporary directory backed by user storage. Looks like good addition to the userstorage library.

For now I would add something like:

    with Backend(request.param) as backend:
        yield backend.tmpdir()